app/src/localization/en/index.ts

```diff
@@ -33,6 +33,17 @@ const translation = {
   "NetInfo": {
     "NoInternetConnectionTitle": "No internet connection",
     "NoInternetConnectionMessage": "You're unable to access services using BC Wallet or receive credentials until you're back online.\n\nPlease check your internet connection."
+  },
```

Comment:
Able to add these place holders in EN to Fr and Br?

Reply:
Already in Bifold. These override the en wording for BC Wallet. Which brings me to the question, does BC Wallet specifically have multi-language support? If so then we need to outright create translation files for fr and pt-br. If not, then we need to override the language screens.